server/impl/meetup.go

```diff
@@ -150,7 +150,7 @@ func (i *Implementation) GetMeetupID(params operations.GetMeetupIDParams) middle
 	if id := SessionFromContext(ctx).Values[UserID]; id == nil {
```

Comment:
If you could do me a favor and rename `idStr` to `userID`, that'd be very appreciated.

Reply:
`userID` is already being used in some places though